splinterd/src/config/builder.rs

```diff
@@ -45,6 +45,7 @@ impl ConfigBuilder {
         }
     }
 
+    #[cfg(feature = "default")]
```

Comment:
Yes, since the main function doesn't actually add any PartialConfig objects so this function is unused without the default features on. However, this just comes up as a warning, so if those are deemed ok, then no, it's not.